docs/src/examples/notebooks/Working_with_Statistics.ipynb

```diff
@@ -373,53 +373,8 @@
     }
    ],
    "source": [
-    "# Load a geojson of an area around Mahebourg, Mauritius\n",
-    "mahebourg = geojson.loads(\"\"\"\n",
-    "{\n",
-    "  \"type\": \"FeatureCollection\",\n",
-    "  \"features\": [\n",
-    "    {\n",
-    "      \"type\": \"Feature\",\n",
-    "      \"properties\": {},\n",
-    "      \"geometry\": {\n",
-    "        \"coordinates\": [\n",
-    "          [\n",
-    "            [\n",
-    "              57.70358910197049,\n",
-    "              -20.384114558699935\n",
-    "            ],\n",
-    "            [\n",
-    "              57.68564920588395,\n",
-    "              -20.384114558699935\n",
-    "            ],\n",
-    "            [\n",
-    "              57.68209507552771,\n",
-    "              -20.39855066753664\n",
-    "            ],\n",
-    "            [\n",
-    "              57.68666467170024,\n",
-    "              -20.421074640746554\n",
-    "            ],\n",
-    "            [\n",
-    "              57.70341985766697,\n",
-    "              -20.434397129770545\n",
-    "            ],\n",
-    "            [\n",
-    "              57.72999121319131,\n",
-    "              -20.42392955694521\n",
-    "            ],\n",
-    "            [\n",
-    "              57.70358910197049,\n",
-    "              -20.384114558699935\n",
-    "            ]\n",
-    "          ]\n",
-    "        ],\n",
-    "        \"type\": \"Polygon\"\n",
-    "      }\n",
-    "    }\n",
-    "  ]\n",
-    "}\n",
-    "\"\"\")\n",
+    "with open(\"data/mahebourg.geojson\", \"r\") as f:\n",
+    "    mahebourg = geojson.load(f)\n",
```

Comment:
any reason to use geojson at all? it seems we are only using load and dumps

Reply:
hmmm, good point. In this case, I will close this PR and make a new one!